cmds/boot/boot.go

```diff
@@ -363,6 +363,16 @@ func kexecLoad(grubConfPath string, grub []string, mountPoint string) error {
 	}
 	// defer ramfs.Close()
 
+	// if /tmp/rsdp file exist we must get the value and add it to the be.cmdline parameter
+	// this will allow the kernel to properly read the ACPI table
+
+	rsdp, err := ioutil.ReadFile("/tmp/rsdp")
+	if err == nil {
```

Comment:
do you need to also add a space between the two strings? I would expect it's a good caution.